js/scripts.js

```diff
@@ -0,0 +1,19 @@
+var span = $("span");
```

Comment:
at first always check that DOM is loaded, and when it will be done, run your code